### PR TITLE
feat: make homepage

### PR DESCRIPTION
**Pull request by zsxlver**
So I have made the homepage using Chara UI and will fix bugs later.
Please comment down the bugs.

### CODE REVIEW

**Review by carrotfarmer — CHANGES_REQUESTED** [review]
image

make sure you export navbar from the component file

File: src/components/HomePage.jsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                Make School Easier<br/>With PiSpace
+            </Text>
+
+            <Text
+              position='absolute'
+              left='0%'
+              right='40.34%;'
+              top='39.51%'
+              bottom='22.3%'
+              fontFamily='Open Sans'
```
Comment (carrotfarmer) [inline src/components/HomePage.jsx]: Also, extending the theme and adding fonts to their respective elements might be better, though we can do it later. https://chakra-ui.com/community/recipes/using-fonts

File: src/Home.jsx [hunk 2]
```diff
@@ -37,9 +39,10 @@ const Home = () => {
         </Center>
       ) : (
         <Center pt="5">
-          <Text>
-            no one's logged in rn. <Link href="/login">sign in</Link>
-          </Text>
+          {/* navbar: */}
+          <Navbar></Navbar>
```
Comment (carrotfarmer) [inline src/Home.jsx]: `<Navbar />`

File: src/Home.jsx [hunk 3]
```diff
@@ -37,9 +39,10 @@ const Home = () => {
...
+          {/* homepage: */}
+          <HomePage></HomePage>
```
Comment (carrotfarmer) [inline src/Home.jsx]: `<HomePage />`

File: src/components/HomePage.jsx (hunk 1 above)
Comment (zsxlver) [reply src/components/HomePage.jsx]: > Also, extending the theme and adding fonts to their respective elements might be better, though we can do it later. https://chakra-ui.com/community/recipes/using-fonts

Yes, I'll do that later

File: src/Home.jsx [hunk 4]
```diff
@@ -1,8 +1,10 @@
 import React from "react";
-import { Button, Center, Heading, Link, Spinner, Text } from "@chakra-ui/react";
+import { Box, Button, Center, Heading, Link, Spinner, Text } from "@chakra-ui/react";
 import { useAuthState } from "react-firebase-hooks/auth";
 import { auth } from "../firebase.config";
 import { signOut } from "firebase/auth";
+import { HomePage } from "./components/HomePage"
+import { Navbar } from "./components/Navbar"
```
Comment (carrotfarmer) [inline src/Home.jsx]: Here you have imported `HomePage` and `Navbar` as a named import but you have exported it has a default export, so do this:
`import HomePage from "./components/HomePage";
`import Navbar from "./components/Navbar";

**Comment by zsxlver** [comment]
AHH gonna start working on this again.

**Comment by carrotfarmer** [comment]
closing due to inactivity

**Comment by zsxlver** [comment]
💀💀

**Comment by carrotfarmer** [comment]
bad boy

**Pull request closed without merging**